decidim-core/app/uploaders/decidim/homepage_image_uploader.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+module Decidim
+  # This class deals with uploading hero images to ParticipatoryProcesses.
+  class HomepageImageUploader < ApplicationUploader
```

Comment:
@josepjaume should we reisze the image? My take is that we shouldn't because we'd rather have it fill as much space as possible without resizing it via the browser, but... /shrug